Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -950,15 +950,15 @@ private void setIRSensitivity()
       }
     });
 
-    int ir_yaw = ini.getInt(Settings.SECTION_CONTROLS, SettingsFile.KEY_WIIBIND_IR_YAW, 15);
+    int ir_yaw = ini.getInt(Settings.SECTION_CONTROLS, SettingsFile.KEY_WIIBIND_IR_YAW, 25);
 
     TextView text_slider_value_yaw = view.findViewById(R.id.text_ir_yaw);
     TextView units_yaw = view.findViewById(R.id.text_ir_yaw_units);
     SeekBar seekbar_yaw = view.findViewById(R.id.seekbar_width);
 
     text_slider_value_yaw.setText(String.valueOf(ir_yaw));
     units_yaw.setText(getString(R.string.yaw));
-    seekbar_yaw.setMax(100);
+    seekbar_yaw.setMax(360);
```

Comment:
This isn't necessarily a good idea. While the value *can* go this high, making it go this high is basically useless, and any increase of the max value makes the setting harder to edit since we're using sliders.

Reply:
I haven't messed around with these settings too much. Should I just leave this value at 100?

Reply:
Yeah, I think that would make the most sense for now.

Reply:
Done